ocp_resources/machine_config_pool.py

```diff
@@ -7,3 +7,30 @@ class MachineConfigPool(Resource):
     class Status(Resource.Status):
         UPDATED = "Updated"
         UPDATING = "Updating"
+
+    def __init__(
+        self,
+        name,
+        labels,
+        machine_config_selector_matches,
+        configuration=None,
+    ):
+        super().__init__(name=name)
+        self.resource_labels = labels
+        self.configuration = configuration
+        self.machine_config_selector_matches = machine_config_selector_matches
+
+    def to_dict(self):
+        super().to_dict()
```

Comment:
Check how this is done in other resources, need to handle yaml_file

Reply:
Why must it handle yaml_file if I'm not going to send a yaml file to this class?

Reply:
Maybe you are not going to send any YAML, but this resource is for everyone to use not just for your use case.

Reply:
I don't see where this was resolved

Reply:
Sorry, I didn't notice I didn't handle yaml_file here in to_dict().